app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  # Method to update the likes counter for a post
+  after_create :update_post_likes_counter
+
+  private
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
Kindly implement this required changes from the previous reviewer. It is yet to be implemented. Kindly check the [differences between after_create and after_save](https://www.altcademy.com/questions/after-create-vs-after-save-in-rails#:~:text=after_create%20%2D%20works%20once%2C%20only%20after,create%20and%20ActiveRecord%23update%20method.) as explained to you by the previous reviewer.